easybuild/easyblocks/generic/cargo.py

```diff
@@ -31,19 +31,20 @@
 @author: Alexander Grund (TU Dresden)
 """
 
+import glob
 import os
 import re
 from collections import defaultdict
 
 import easybuild.tools.environment as env
 import easybuild.tools.systemtools as systemtools
-from easybuild.tools.build_log import EasyBuildError, print_warning
 from easybuild.framework.easyconfig import CUSTOM
 from easybuild.framework.extensioneasyblock import ExtensionEasyBlock
-from easybuild.tools.filetools import extract_file
-from easybuild.tools.run import run_cmd
+from easybuild.tools.build_log import EasyBuildError, print_warning
 from easybuild.tools.config import build_option
-from easybuild.tools.filetools import compute_checksum, mkdir, move_file, read_file, write_file, CHECKSUM_TYPE_SHA256
+from easybuild.tools.filetools import CHECKSUM_TYPE_SHA256, compute_checksum, extract_file, mkdir, move_file
+from easybuild.tools.filetools import read_file, write_file
```

Comment:
Maybe se a single import?
```suggestion
from easybuild.tools.filetools import (CHECKSUM_TYPE_SHA256, compute_checksum, extract_file, mkdir, move_file,
                                       read_file, write_file)
```

Reply:
For consistency across most EasyBuild code, I would prefer not taking this approach (but it's a detail)